integrations/azure_ai_search/pyproject.toml

```diff
@@ -0,0 +1,164 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "azure-ai-search-haystack"
+dynamic = ["version"]
+description = 'Haystack 2.x Document Store for Azure AI Search'
+readme = "README.md"
+requires-python = ">=3.8,<3.13"
```

Comment:
3.8 already reached End Of Life, so for new integrations, I would drop it here and in test matrices